web-client/public/js/generateNetwork.js

```diff
@@ -151,22 +179,21 @@ containing "-", "_", and alpha-numeric characters only`);
             alert(`GRNsight is only capable of handling 75 genes at most. Your proposed network contains
  ${genesAmount} genes. Please remove some genes from your proposed network.`);
         } else {
-
-            let source = grnState.customWorkbook.source;
-            let headers = {
-                type:"CreateNetwork",
-                info: {
-                    genes: grnState.customWorkbook.genes,
-                    source:grnState.customWorkbook.sources[source].source,
-                    timestamp:grnState.customWorkbook.sources[source].timestamp
-                }
+            const genes = Object.keys(grnState.customWorkbook.genes).map(g => {
+                return grnState.customWorkbook.genes[g];
+            });
```

Comment:
Interesting that lint gets mad here. This may be a setting

Reply:
It was indeed a lint setting. I updated the `.eslintrc.yaml` file and changed the settings